Priyaa/HTML Basics Remake/Assignment-1.html

```diff
@@ -1,49 +1,60 @@
 
-<img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" width="250" , height="200" />
+   <p><b>Merlo John Pusey</b>(February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington, D.C.)[1][2] was an
+      <u>American biographer</u> and <u>editorial writer</u>. He won the 1952 Pulitzer Prize for Biography or
+      Autobiography
+      and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.<br></p>
 
-<p><b>Merlo John Pusey</b>(February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington, D.C.)[1][2] was an <u>American biographer</u> and <u>editorial writer</u>. He won the 1952 Pulitzer Prize for Biography or Autobiography
-        and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.<br></p>
-        
-        <p>Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college newspaper. He later became
-        a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]<br></p>
-        
-       <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946, continuing to contribute occasional pieces until about two years before his death.
-        <br></p>
-        <p><b>Contents</b></p>
-        <p><a href="Books.html">1. Books</a><br>
-           <a href="References.html">2. References</a></p>
+   <p>Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints
+      University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college
+      newspaper. He later became
+      a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]<br></p>
 
-           <h2>Books</h2>
-           <hr>
-           <ul>
-              <li>The Supreme Court Crisis(Macmillan, 1937)</li>
-              <li>Big Government: Can We Control It? (Harper, 1945)</li> 
-              <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
-               <li> Eisenhower, the President (Macmillan, 1956)</li></p>
-            </ul>
+   <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946,
+      continuing to contribute occasional pieces until about two years before his death.
+      <br></p>
+   <p><b>Contents</b></p>
+   <p><a href="Books.html">1. Books</a><br>
+      <a href="References.html">2. References</a></p>
 
+   <h2>Books</h2>
+   <hr>
+   <ul>
+      <li>The Supreme Court Crisis(Macmillan, 1937)</li>
+      <li>Big Government: Can We Control It? (Harper, 1945)</li>
+      <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
+      <li> Eisenhower, the President (Macmillan, 1956)</li>
+      </p>
+   </ul>
```

Comment:
Use `div` with `id` to link this section with `a` tag you used in the content section.